mobile/src/calendar/event-dialog/confirm-sick-leave-event-dialog.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import { EventDialogBase, eventDialogTextDateFormat } from './event-dialog-base';
+import { AppState } from '../../reducers/app.reducer';
+import { Dispatch } from 'redux';
+import { connect } from 'react-redux';
+import { EventDialogActions, closeEventDialog, openEventDialog } from '../../reducers/calendar/event-dialog/event-dialog.action';
+import { DayModel } from '../../reducers/calendar/calendar.model';
+import { Employee } from '../../reducers/organization/employee.model';
+import { confirmSickLeave } from '../../reducers/calendar/sick-leave.action';
+import { Moment } from 'moment';
+import { EventDialogType } from '../../reducers/calendar/event-dialog/event-dialog-type.model';
+
+interface ClaimSickLeaveEventDialogDispatchProps {
+    back: () => void;
+    confirmSickLeave: (employeeId: string, startDate: Moment, endDate: Moment) => void;
+    closeDialog: () => void;
+}
+
+interface ClaimSickLeaveEventDialogProps {
+    startDay: DayModel;
+    endDay: DayModel;
+    userEmployee: Employee;
+}
+
+class ConfirmSickLeaveEventDialogImpl extends Component<ClaimSickLeaveEventDialogProps & ClaimSickLeaveEventDialogDispatchProps> {
+    public render() {
+        return <EventDialogBase 
+                    title={'Select date to Complete your Sick Leave'} 
+                    text={this.text}
+                    icon={'sick_leave'} 
+                    cancelLabel={'Back'}
+                    acceptLabel={'Confirm'}
+                    acceptAction={this.acceptAction}
```

Comment:
maybe rename to onAcceptClick onCancelClick onCloseClick?

Reply:
Press :)